base/src/test/java/io/fabric8/launcher/base/identity/IdentityFactoryTest.java

```diff
@@ -4,6 +4,9 @@
 import org.junit.Rule;
```

Comment:
Probably this class name needs to change since IdentityFactory is gone

Reply:
right !!